### ISSUE TITLE
nalgebra-glm support for serde serialisation?

### ISSUE THREAD

**Issue by repi** [rustsim/nalgebra#513]
Been using the `serde-serialize` feature of nalgebra but ran into an issue with nalgebra-glm which doesn't use that feature of nalgebra nor has a feature toggle available for it. So one can't serialize any of the nalgebra-glm types.

I find that most of my use cases of the math types does indeed need serialization support, could nalgebra-glm be changed to include `serde-serialize`? This is what I've done locally in a fork right now. 

Though ideally one should have the same feature toggle on it also, but not sure if one can do features in crates that enables features in other crates?

**Comment by sebcrozet** [comment]
Yes, nalgebra-glm should be changed to include the `serde-serialize` feature. It is possible for a feature to toggle one of the dependencies features too, this basically looks like this:

```toml
[features]
serde-serialize = [ "nalgebra/serde-serialize" ]
```

You will find another example on the rust book [there](https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section) (see the comment regarding the "session" feature).

**Comment by sebcrozet** [comment]
This is released in nalgebra-glm v0.2.1

**Comment by repi** [comment]
Thanks!
